pkg/kubectl/cmd/util/helpers.go

```diff
@@ -302,6 +302,12 @@ func UsageErrorf(cmd *cobra.Command, format string, args ...interface{}) error {
 	return fmt.Errorf("%s\nSee '%s -h' for help and examples.", msg, cmd.CommandPath())
 }
 
+func HelpErrorf(cmd *cobra.Command, format string, args ...interface{}) error {
```

Comment:
I don't think we need to wrap it as a new method.

Reply:
i think when check ```args``` occur err, print help() info very reasonable. this will reduce repetition code

Reply:
If kubectl is going to include more utilities, I'd expect them to be used way more places than just the two sections replaced here.

Reply:
updated

Reply:
> I don't think we need to wrap it as a new method.

+1, I struggle to see the valid in this. We're trying to reduce the amount of utils external to core packages, not increase. So I'd say make it private in `pkg/kubectl/cmd/config` and use it in the `config ...` commands.